train/config.json

```diff
@@ -21,7 +28,15 @@
         "latest_data_entries": 16384,
         "iteration_data_entries": 1024,
         "batch_size": 32,
+        "learning_rate": [
+            [40960, 0.01],
+            [61440, 0.001],
+            [       0.0001]
+        ],
         "compare": {
+            "temperature_policy": [
+                [       0.0]

```

Comment:
Doesn't this cause all the games to be the same? We can different games to be compared

Reply:
Well shit...
Dir noise?
If we want to compare them in an accurate way we should avoid both temperature and Dir noise. The paper says temperature is zero but nothing about the noise.
Maybe a better approch will be to use random position (no idea how to generate) with no random at all

Reply:
Maybe they can both play with high noise/temperature only in the first few turns (very few, e.g. 3), then the rest of the game is 0 noise/temp

Reply:
Also I think we should add a consistency (no-random) test in the case of temperature=noise=0

Reply:
What noise??

![image](https://user-images.githubusercontent.com/4618146/197379637-5ad8348e-d8a6-4bd9-9af9-48f7054f5993.png)